parse/parser.go

```diff
@@ -178,8 +178,18 @@ func Parse(buf TokenBuffer) (*ast.Program, error) {
 // TODO: implement me w/ test cases :-)
 func parseStatement(buf TokenBuffer) (ast.Statement, error) {
 	switch buf.Peek(CURRENT).Type {
+	case Assign:
```

Comment:
I think Assign should be dealt inside the parseAssignStatement, so I think this is not needed

Reply:
But without this, there was an error.
 I will retry.